include/libvfio-user.h

```diff
@@ -261,6 +286,13 @@ vfu_setup_region(vfu_ctx_t *vfu_ctx, int region_idx, size_t size,
                  struct iovec *mmap_areas, uint32_t nr_mmap_areas,
                  int fd);
 
+/*
+ * Returns the size of the migration registers, which is guaranteed to be page
```

Comment:
This makes it sound like the registers themselves take up a page. Maybe generalize a bit?

Returns the size of the area needed to hold the migration registers at the beginning of the migration region; guaranteed to be page aligned.

and call it "vfu_get_migr_register_area_size" or somesuch? What do you think?

Reply:
ok